demisto_sdk/commands/download/downloader.py

```diff
@@ -95,647 +87,851 @@
         keep_empty_folders: bool = False,
         auto_replace_uuids: bool = True,
         **kwargs,
     ):
         self.output_pack_path = output
-        self.input_files = [input] if isinstance(input, str) else input
+        self.input_files = [input] if isinstance(input, str) else list(input)
         self.regex = regex
         self.force = force
-        self.download_system_item = system
+        self.download_system_items = system
         self.system_item_type = item_type
         self.insecure = insecure
         self.list_files = list_files
-        self.all_custom_content = all_custom_content
+        self.download_all_custom_content = all_custom_content
         self.run_format = run_format
         self.client = None
-        self.custom_content_temp_dir = mkdtemp()
-        self.system_content_temp_dir = mkdtemp()
-        self.all_custom_content_objects: List[dict] = list()
-        self.files_not_downloaded: List[list] = list()
-        self.custom_content: List[dict] = list()
-        self.pack_content: Dict[str, list] = {
-            entity: list() for entity in CONTENT_ENTITIES_DIRS
-        }
-        self.num_merged_files = 0
-        self.num_added_files = 0
         self.init = init
         self.keep_empty_folders = keep_empty_folders
         self.auto_replace_uuids = auto_replace_uuids
         if is_sdk_defined_working_offline() and self.run_format:
             self.run_format = False
-            logger.info(
-                "format is not supported when the DEMISTO_SDK_OFFLINE_ENV environment variable is set, skipping it."
+            logger.warning(
+                "Formatting is not supported when 'DEMISTO_SDK_OFFLINE_ENV' environment variable is set.\n"
+                "Downloaded files will not be formatted."
             )
 
     def download(self) -> int:
         """
-        Downloads custom content data from Demisto to the output pack in content repository.
-        :return: The exit code
-        """
-        exit_code: int = self.download_manager()
-        self.remove_traces()
-        return exit_code
+        Downloads content items (system or custom) from XSOAR / XSIAM to the provided output path.
 
-    def download_manager(self) -> int:
-        """
-        Manages all download command flows
-        :return The exit code of each flow
+        Returns:
+            int: Exit code. 1 if failed, 0 if succeeded
         """
-        if not self.verify_flags():
-            return 1
-        if not self.download_system_item and not self.fetch_custom_content():
-            return 1
-        if self.download_system_item and not self.fetch_system_content():
-            return 1
-        if self.handle_list_files_flag():
-            return 0
-        self.handle_init_flag()
-        self.handle_all_custom_content_flag()
-        self.handle_regex_flag()
-        if not self.verify_output_pack_is_pack():
-            return 1
+        try:
+            if self.list_files:
```

Comment:
These names correspond to the values provided by `click` (which correspond to the name of the flags - in this case `--list-files`). For example, in `__init__` we will have `self.should_list_all_custom_content = list_files`.
Are you sure you want me to change these? I think it might cause confusion.